lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.spec.ts

```diff
@@ -454,6 +485,54 @@ describe('AttachFileCloudWidgetComponent', () => {
             expect(openUploadFileDialogSpy).toHaveBeenCalledWith('-my-', 'multiple', true, true);
         });
 
+        it('Should set default user alias (-my-) as rootNodeId if destinationFolderPath does not defined', async () => {
+            const getAliasAndPathSpy = spyOn(widget, 'getAliasAndRelativePathFromDestinationFolderPath').and.callThrough();
+            widget.field = new FormFieldModel(new FormModel(), {
+                type: FormFieldTypes.UPLOAD,
+                value: []
+            });
+            widget.field.id = 'attach-file-alfresco';
+            widget.field.params = <FormFieldMetadata> allSourceWithoutDestinationFolderPath;
+            widget.field.params.multiple = true;
```

Comment:
is it required for the purpose of this test to make the widget multiple?

Reply:
its not required, Removed it. Thanks